.github/workflows/codespell.yml

```diff
@@ -7,7 +7,7 @@ concurrency:
 jobs:
   codespell:
     name: CodeSpell
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04-arm
```

Comment:
Minor: Why not ubuntu-24.04-arm?

The difference between sensible packages (e.g. openssl) is negligible between 22.04 and 24.04

Reply:
Mainly because we are currently running CI against Ubuntu 22.

Reply:
Isn’t the `-latest` same as 24?

Reply:
According to https://docs.github.com/en/actions/using-github-hosted-runners/using-github-hosted-runners/about-github-hosted-runners#standard-github-hosted-runners-for-public-repositories, there is an `ubuntu-latest`, `windows-latest`, and `macos-latest`, but no `-latest` are documented for the arm runners.

Reply:
I mean, before we switched to arm runners, we used -latest, which is ubuntu 24.
But now, we switched to arm runners and ubuntu 22. Even though ubuntu 24 is available for arm, too. Why?

Reply:
Huh, that’s interesting. Our CI runs (e.g. [this](https://github.com/rubocop/rubocop-rspec/actions/runs/12870100230)) has an annotation at the bottom saying

> ubuntu-latest pipelines will use ubuntu-24.04 soon. For more details, see https://github.com/actions/runner-images/issues/10636

I understood that as “CI will run ubuntu 24, but _not yet_”. But looking closer at any CI run, I of course now see `Image: ubuntu-24.04`. So yes of course, this PR should target Ubuntu 24 as well.

Reply:
Ha! Understood. I am sorry I didn’t when you first explained.
No objections then!